app/scripts/modules/netflix/fastProperties/fastProperty.read.service.ts

```diff
@@ -0,0 +1,36 @@
+import { module, IPromise } from 'angular';
+import {API_SERVICE, Api} from 'core/api/api.service';
+import {Scope} from './domain/scope.domain';
+
+export class FastPropertyReaderService {
+
+  static get $inject() {
+    return ['API'];
+  }
+
+  constructor(private API: Api) {}
+
+  public fetchForAppName(appName: string): IPromise<any> {
+    return this.API.all('fastproperties').all('application').one(appName).get();
+  }
+
+  public search(searchTerm: string): IPromise<any> {
+    return this.API.all('fastproperties').all('search').one(searchTerm).get();
+  }
+
+  public getPropByIdAndEnv(id: string, env: string): IPromise<any> {
+    return this.API.all('fastproperties').one('id', id).one('env', env).get();
+  }
+
+  public fetchImpactCountForScope(fastPropertyScope: Scope): IPromise<any> {
+    return this.API.all('fastproperties').all('impact').post(fastPropertyScope);
+  }
+
+}
+
+export const FAST_PROPERTY_READER_SERVICE = 'spinnaker.netflix.fastProperties.read.service';
```

Comment:
My preference would be `FAST_PROPERTY_READ_SERVICE` or `FAST_PROPERTY_READER` here, but I can live with this.